contracts/introspection/ERC165.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.23;
+
+
+/**
+ * @title ERC165
+ * @dev https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md
+ */
+interface ERC165 {
+
+  /**
+   * @notice Query if a contract implements an interface
+   * @param _interfaceId The interface identifier, as specified in ERC-165
+   * @dev Interface identification is specified in ERC-165. This function
+   * @dev  uses less than 30,000 gas.
```

Comment:
Should we add a test to check gas usage? While the current lookup implementation will always be below the limit, I feel this would be nice to have for completeness, and to safeguard against future changes to said implementation.

Reply:
done!